app/controllers/comments_controller.rb

```diff
@@ -1,25 +1,42 @@
 class CommentsController < ApplicationController
+  load_and_authorize_resource
```

Comment:
- [ ] Kindly remove the `load_and_authorize_resource` as it causes an error when I try to create a new comment.